README.MD

```diff
@@ -1,8 +1,137 @@
-#Test runner
+# Jarvis Test Runner
 
-###Test BQ view
+## Package
+To package the test runner application into a distributable JAR run 
+```
+mvn clean package spring-boot:repackage
+```
+in the command line, then to execute use
+```
+java -jar target/name-of-your-jar.jar "arg1" "arg2"
+```
+
+## Context
+A context represents how the test runner can reach local and external resources that are necessary for the tests. 
+Contexts are parsed from the __contexts.json__ file in the __test\resources__ folder by the ContextLoader.
```

Comment:
the path to the context file should be an input argument for the application. 
You can use the --context argument name

Reply:
Done.